packages/bcer-api/app/src/location/location.service.ts

```diff
@@ -458,7 +458,8 @@ export class LocationService {
         await queryRunner.manager.delete('location_manufactures_manufacturing', { locationId });
         await queryRunner.manager.delete('location_products_product', { locationId });
         await queryRunner.manager.delete('salesreport', { locationId });
-        await queryRunner.manager.delete('product_sold', { locationId });
```

Comment:
@DRsabadash this line here. somehow typeorm was shouting that locationId column does not exist even though it definitely exists. I guess it is throwing an error because the Entity did not have the locationId property :'(